src/content_script/index.tsx

```diff
@@ -22,9 +22,16 @@ async function popupThumbClickHandler(event: MouseEvent) {
     if (!$popupThumb) {
         return
     }
-    const x = $popupThumb.offsetLeft
-    const y = $popupThumb.offsetTop
-    showPopupCard(x, y, $popupThumb.dataset['text'] || '')
+
+    const range = window.getSelection()?.getRangeAt(0)
+    const rect = range?.getBoundingClientRect()
+    const popupWidth = 600
```

Comment:
Where does this magic number come from and can it be used as a global variable?

Reply:
This is the width of the card that I obtained from inspecting elements, perhaps we have a better variable to replace it.